tests/test_field_handler.py

```diff
@@ -6,7 +6,7 @@
                              HasManyDescriptor, HasAndBelongsToManyDescriptor)
 from remodel.utils import create_tables, create_indexes
 
-from . import BaseTestCase, DbBaseTestCase
```

Comment:
I get an error:

      File "/repos/remodel/tests/test_field_handler.py", line 8, in <module>
    from . import BaseTestCase, DbBaseTestCase
SystemError: Parent module '' not loaded, cannot perform relative import